app/models/image.rb

```diff
@@ -84,6 +84,18 @@ class Image < ActiveFedora::Base
     index.as :stored_searchable, :facetable
   end
 
+  property :nul_creator, predicate: ::Vocab::Donut.has_creator, multiple: true do |index|
+    index.as :stored_searchable, :facetable
+  end
+
+  property :nul_subject, predicate: ::Vocab::Donut.has_subject, multiple: true do |index|
```

Comment:
In the spreadsheet it looks like Jen specified a local predicate for creator and contributor but not for subject, she specified a different one for this

Reply:
looks like it would be `::RDF::Vocab::DC.subject`